job-dsls/jobs/new_downstream_production.groovy

```diff
@@ -16,7 +16,7 @@ def final DEFAULTS = [
                 "**/target/kie-server-*ee7.war",
                 "**/target/kie-server-*webc.war",
                 "**/target/jbpm-server*dist*.zip",
-                "**/target/business-monitoring-webapp.war",
+                "**/target/business-monitoring-webapp.war",downsteram.production y upstream
```

Comment:
what is this "y" ?

Reply:
this was a line that accidental was introduced